### PR TITLE
Add fr translation

### PR DESCRIPTION
**Pull request by Tbaut**
Fixed some typos in english along the way.
Pretty sure there're many weird things remaining.. not used to read about blockchain stuff in french :baguette_bread:

### CODE REVIEW

**Comment by jacogr** [comment]
Just leaving it open for @chevdor to take a brief look at. (Doesn't need to be fully-comprehensive, Rome was not built in a day). The fixed to the existing strings are 100%.

File: packages/apps/public/locales/en/app-accounts.json [hunk 1]
```diff
@@ -213,6 +213,7 @@
...
   "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. The \"///password\" is optional and should only occur once.": "You can set a custom derivation path for this account using the following syntax \"/<soft-key>//<hard-key>///<password>\". The \"/<soft-key>\" and \"//<hard-key>\" may be repeated and mixed`. The \"///password\" is optional and should only occur once.",
   "You don't have any accounts. Some features are currently hidden and will only become available once you have accounts.": "You don't have any accounts. Some features are currently hidden and will only become available once you have accounts.",
+  "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app in the navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.": "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app in the navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.",
```
Comment (jacogr) [inline packages/apps/public/locales/en/app-accounts.json:216]: Not sure why this is the same as the line below - seems we have a fix applied that may not reflect everywhere. ... yes, it sucks.

**Comment by chevdor** [comment]
I have dumped my commits. I am sure it does not cover 100% but we get even closer :)
There is an issue: the key "Address book" is missing. cc @jacogr. There may be a few others but since this one is at the root level of the menus, it is easy to spot.

**Review by chevdor — APPROVED** [review]

File: packages/apps/public/locales/fr/translation.json [hunk 2]
```diff
@@ -600,26 +600,26 @@
...
   "The account's password specified at the creation of this account.": "Le mot de passe du compte spécifié lors de la création de ce compte.",
   "The actual JSONRPC module and function to make a call to.": "Le module JSONRPC réel et la fonction vers laquelle effectuer un appel.",
   "The actual account you wish to submit the bid with": "Le compte réel avec lequel vous souhaitez soumettre l'enchère",
   "The actual fully constructed signed output. This can be used for submission via other channels.": "La sortie signée réellement construite. Cela peut être utilisé pour la soumission via d'autres canaux.",
   "The actual proposal to make, based on the selected call and parameters thereof.": "La proposition réelle à faire, basée sur l'appel sélectionné et ses paramètres.",
   "The address for the deployed contract instance.": "L'adresse de l'instance de contrat déployée.",
-  "The address index (derivation on account) to use": "L'index d'adresse (dérivation sur compte) pour utiliser",
+  "The address index (derivation on account) to use": "L'indexe d'adresse (dérivation sur compte) à utiliser",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:622]:
```suggestion
  "The address index (derivation on account) to use": "L'index d'adresse (dérivation sur compte) à utiliser",
```
It's like a database index https://fr.wikipedia.org/wiki/Index_(base_de_donn%C3%A9es)

File: packages/apps/public/locales/fr/translation.json [hunk 3]
```diff
@@ -950,7 +950,7 @@
   "address created": "adresse créée",
   "address edited": "adresse modifiée",
   "address forgotten": "adresse oubliée",
-  "address index": "index d'adresses",
+  "address index": "indexe d'adresse",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:955]:
```suggestion
  "address index": "index d'adresse",
```

File: packages/apps/public/locales/fr/translation.json [hunk 4]
```diff
@@ -1340,8 +1340,8 @@
   "tags": "label",
   "target": "target",
   "term progress": "term progress",
-  "the account to make recoverable": "the account to make récupable",
-  "the account to recover to": "the account pour récupérer vers",
+  "the account to make recoverable": "le compte a rendre récupérable",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:1346]:
```suggestion
  "the account to make recoverable": "le compte à rendre récupérable",
```

File: packages/apps/public/locales/fr/translation.json [hunk 5]
```diff
@@ -0,0 +1,1465 @@
...
+  "Multiple council proposals could exist, both approval and rejection. Apply your vote to the correct council proposal (also available on council motions page)": "Plusieurs propositions du conseil pourraient exister, à la fois approbation et rejet. Appliquez votre vote à la bonne proposition du conseil (également disponible sur la page des motions du conseil)",
+  "Multisig": "Multisig",
+  "Multisig approval with hash (non-final approval)": "Approbation multisig avec hash (approbation non finale)",
+  "Multisig approval with hash (not message with call)": "Approbation multisig avec hash (pas de message avec appel)",
+  "Multisig approvals": "Approbations multisig",
+  "Multisig approvals pending": "Approbations multisig en cours",
+  "Multisig message with call (for final approval)": "Message multisig avec appel (pour approbation finale)",
+  "Multisig message with call (not approval with hash)": "Message multisig avec appel (pas d'approbation avec hachage)",
+  "Mutates contract state": "Mutate l'état du contrat",
+  "My On-Chain Name": "Mon nom sur la Blockchain",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:299]:
```suggestion
  "My On-Chain Name": "Mon nom sur la blockchain",
```

File: packages/apps/public/locales/fr/translation.json [hunk 6]
```diff
@@ -0,0 +1,1465 @@
...
+  "only this network": "uniquement ce réseau",
+  "only with an identity": "avec une identité",
+  "origin": "origine",
+  "other": "autre",
+  "other stake": "autre enjeu",
+  "our best": "notre meilleur",
+  "overall/validator": "global / validateur",
+  "own": "propre",
+  "own stake": "propre investissement",
+  "parachain id": "parachain id",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:1195]: for consistency
```suggestion
  "parachain id": "Para-chain id",
```

File: packages/apps/public/locales/fr/translation.json [hunk 7]
```diff
@@ -0,0 +1,1465 @@
...
+  "Delegate democracy votes": "Déléguer les votes pour Démocratie",
+  "Delete this custom example": "Supprimer cet exemple personnalisé",
+  "Democracy overview": "Résumé de la démocratie",
+  "Denomination poll": "Sondage sur la dénomination",
+  "Deploy": "Déployer",
+  "Deploy a code hash": "Déployer un hachage de code",
+  "Deploy this code hash as a smart contract": "Déployer ce hash de code en tant que Smart Contract",
+  "Deploy with this constructor": "Déployer avec ce constructeur",
+  "Deposit": "Déposer",
+  "Deregister this parachain": "Annuler l'enregistrement de cette parachain",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:184]:
```suggestion
  "Deregister this parachain": "Annuler l'enregistrement de cette Para-chain",
```

File: packages/apps/public/locales/fr/translation.json [hunk 8]
```diff
@@ -0,0 +1,1465 @@
...
+  "nav.council": "Counseil",
+  "nav.dashboard": "Tableau de bord",
+  "nav.democracy": "Démocracie",
+  "nav.explorer": "Explorateur",
+  "nav.extrinsics": "Extrinsics",
+  "nav.generic-asset": "Objet génériques",
+  "nav.github": "Github",
+  "nav.i18n": "Traduction",
+  "nav.js": "Javascript",
+  "nav.parachains": "Parachains",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:1156]:
```suggestion
  "nav.parachains": "Para-chains",
```

File: packages/apps/public/locales/fr/translation.json [hunk 9]
```diff
@@ -1357,7 +1357,7 @@
   "the stash account to nominate with": "le compte caché avec lequel désigner",
   "the supplied signature": "la signature fournie",
   "threshold": "seuil",
-  "tip": "pourboire",
+  "tip": "récompense",
```
Comment (Tbaut) [inline packages/apps/public/locales/fr/translation.json:1360]: We should use one or the other everywhere, right now we have a mix, with "récompense" also being used for block reward. I'm not a fan of "pourboire" for tip either :man_shrugging: .

**Review by Tbaut — COMMENTED** [review]
Thank yo so much for that. It's a lot of work!
I've found a couple inconsistencies like for parachains. I wonder if we keep "blockchain" as is, then we should keep parachain as is a well, rather than translating and capitalizing it with "Para-chaine".

edit: changes it after our talk about it.

File: packages/apps/public/locales/fr/translation.json (hunk 9 above)
Comment (Tbaut) [reply packages/apps/public/locales/fr/translation.json:1360]: Ok I'll leave that as is.

**Comment by Tbaut** [comment]
:heavy_check_mark:  I can't approve my own PR, but I'm good with that, thanks a lot @chevdor

File: packages/apps/public/locales/en/app-accounts.json (hunk 1 above)
Comment (jacogr) [reply packages/apps/public/locales/en/app-accounts.json:216]: Adjusted in https://github.com/polkadot-js/apps/pull/3829

**Comment by jacogr** [comment]
Thank you!

**Pull request merged**